### ISSUE TITLE
Release QtSASS 0.3.1

### ISSUE THREAD

**Issue by vToMy** [spyder-ide/qtsass#60]
The current latest version doesn't work with python3.
Master does work.

**Comment by s-weigand** [comment]
I guess the main issue is with python 3.10 since I get this warning in `pytest`
```python
qtsass\api.py:16: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
    from collections import Mapping, Sequence
```
@ccordoba12 @goanpeca It would be awesome if we can get a release that works with py310 😄 

For now my workaround is putting the following lines is my `setup.cfg`
```ini
[options]
install_requires =
    qtsass>=0.3.0 ;python_version < '3.10'
    qtsass@https://github.com/spyder-ide/qtsass/archive/refs/heads/master.zip ;python_version >= '3.10'
```

**Comment by dpizetta** [comment]
Hey guys, I think it is a good idea to provide a new release with the changes that were implemented so far. Tks

**Comment by s-weigand** [comment]
While my workaround does the job locally, it blocks me from releasing my package to PyPI 😥
![image](https://user-images.githubusercontent.com/9513634/188104439-301a3e23-3b94-4dc2-98cd-3a988f51f31f.png)

**Comment by vToMy** [comment]
pinging @goanpeca

**Comment by s-weigand** [comment]
As a workaround, I published the [current state of `qtsass`](https://github.com/spyder-ide/qtsass/tree/06f15194239ba430d5a9a144c1cc7c6b03e585a3) as [qtsass310](https://pypi.org/project/qtsass310/) on PyPI.

**Comment by ccordoba12** [comment]
Hey guys, sorry for the delay in answering your requests. @dalthviz will take care of releasing a new version (0.3.1) shortly that will have the fixes for Python 3.10.
